### ISSUE TITLE
[1.4 Alpha] Steam Multiplayer Not Working

### ISSUE THREAD

**Issue by Solxanich** [tModLoader/tModLoader#1707]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

Latest public-1.4-alpha

### OS

Windows

### Platform

Steam

### Description

Steam multiplayer gets stuck on connecting.

Either on Sending Tile Data or at Found Server.

EDIT Aug 29:
There appears to be two different bugs present here. 
1) The '+connect_lobby ###' argument is no longer detected by LaunchParameters.ContainsKey. This makes local testing difficult
2) The actual connection itself is bugged and is what this issue originally represents. May be related to #1490 . Platform Not Supported. Going to outright guess its relate to WeGame somehow.

### Log File

Reoccurring in Support

### Additional Information

Issue incomplete. Logging for reminder purposes.

IP hosting has been working as an alternative

**Comment by Solxanich** [comment]
'Starting tModLoader from steam, then starting the server from start-tModLoader.bat was the issue. Starting the server first then starting TML using start-tModLoader.bat was the fix.' - another issue WRT Steam and server hosting. 
May or may not be related to the issue of the WorkshopHelper.tml calling SteamAPI.Init() through a reference to a static that is fixed in #1732

**Comment by Solxanich** [comment]
Confirmed fixed after #1812
